telos-frontend/src/views/journal/Page.jsx

```diff
@@ -1,44 +1,26 @@
-import { useState, useCallback, useEffect } from 'react';
+import { useCallback } from 'react';
 import { useDrop } from 'react-dnd';
 import update from 'immutability-helper';
 import DraggableWidget from '../../dnd/DraggableWidget';
 import { WidgetTypes } from '../../dnd/WidgetTypes';
 import pageStyles from './Page.module.css';
 
-const Page = ({ date, leftPage }) => {
-  const [widgets, setWidgets] = useState({});
-
-  useEffect(() => {
-    // Temp code to display all widgets. Remove when backend is ready.
-    if (leftPage) {
-      setWidgets({
-        0: { widgetType: 'todo', top: 0, left: 0 },
-        1: { widgetType: 'habit_tracker', top: 200, left: 200 },
-      });
-    } else {
-      setWidgets({
-        2: { widgetType: 'text', top: 100, left: 100 },
-        3: { widgetType: 'calendar', top: 300, left: 300 },
-      });
-    }
-
-    // TODO: Fetch widgets
-  }, [date]);
-
+// eslint-disable-next-line no-unused-vars
```

Comment:
Maybe add a comment here to explain why this disable is here. I assume that once the unused variables are needed in the program this line will be removed?

Reply:
Yeah I was helping develop part of this and the disables were needed for the time being. I do agree that comment on this for context would be useful.

Reply:
yup, added